build.cmd

```diff
@@ -17,12 +18,13 @@ if not exist %NUGET_CACHE_DIR%\NuGet.exe (
 )
 
 :: copy nuget locally
-if not exist src\.nuget\NuGet.exe (
-  copy %NUGET_CACHE_DIR%\NuGet.exe src\.nuget\NuGet.exe > nul
+if not exist %NUGET_LOCAL_DIR% md %NUGET_LOCAL_DIR%
+if not exist %NUGET_LOCAL_DIR%\NuGet.exe (
+  copy %NUGET_CACHE_DIR%\NuGet.exe %NUGET_LOCAL_DIR%\NuGet.exe > nul
 )
 
 :: restore packages
-src\.nuget\NuGet.exe restore .\src\NServiceBus.RabbitMQ.sln -MSBuildVersion 14
+%NUGET_LOCAL_DIR%\NuGet.exe restore .\packages.config -PackagesDirectory ./src/packages -MSBuildVersion 14 -Verbosity quiet
```

Comment:
I assume the new separate packages.config is to avoid restoring the items that are only used by TeamCity? Seems reasonable, but I'm wondering if there might be a better location/filename for the new file that makes it clearer it is for pulling down packages that the build script needs?